src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/OffenderService.kt

```diff
@@ -87,6 +84,10 @@ class OffenderService(
     }
   }
 
+  private fun hasRestrictionOrExclusion(probationOffenderDetail: ProbationOffenderDetail): Boolean {
+    return probationOffenderDetail.currentExclusion == true || probationOffenderDetail.currentRestriction == true
```

Comment:
Minor..
`return probationOffenderDetail.currentExclusion || probationOffenderDetail.currentRestriction`

Reply:
`currentExclusion` and `currentRestriciton` could be null, so Kotlin doesn't like doing this, the linter suggests adding the `== true`